static/redfish/v1/JsonSchemas/OemAssembly/index.json

```diff
@@ -0,0 +1,45 @@
+{
+    "$id": "http://redfish.dmtf.org/schemas/v1/OemAssembly.v1_0_0.json",
+    "$schema": "http://redfish.dmtf.org/schemas/v1/redfish-schema-v1.json",
+    "copyright": "Copyright 2014-2019 DMTF. For the full DMTF copyright policy, see http://www.dmtf.org/about/policies/copyright",
+    "definitions": {
+        "Assembly": {
+            "additionalProperties": false,
+            "description": "An indication of whether the system is prepared for the assembly to be changed.",
+            "parameters": {},
+            "patternProperties": {
+                "^([a-zA-Z_][a-zA-Z0-9_]*)?@(odata|Redfish|Message)\\.[a-zA-Z_][a-zA-Z0-9_]*$": {
+                    "description": "This property shall specify a valid odata or Redfish property.",
+                    "type": [
+                        "array",
+                        "boolean",
+                        "integer",
+                        "number",
+                        "null",
+                        "object",
+                        "string"
+                    ]
+                }
+            },
+            "properties": {
+                "ReadyToRemove": {
+                    "description": "An indication of whether the system is prepared for the RTC battery to be changed.",
```

Comment:
Please remove any reference to RTC battery and replace with assembly as we talked about. 
This is not an battery schema but an assembly schema